lgc/include/lgc/patch/PatchLoopMetadata.h

```diff
@@ -33,7 +33,6 @@
 #include "lgc/patch/Patch.h"
 #include "lgc/state/PipelineState.h"
 #include "lgc/state/TargetInfo.h"
-//#include "llvm/Analysis/LoopAnalysisManager.h"
```

Comment:
Why was this commented out but not removed?

Reply:
not sure about the reason but since it is commented, it's better to remove it.